pkg/querier/queryrange/roundtrip.go

```diff
@@ -68,6 +71,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.AlignQueriesWithStep, "querier.align-querier-with-step", false, "Mutate incoming queries to align their start and end with their step.")
 	f.BoolVar(&cfg.CacheResults, "querier.cache-results", false, "Cache query results.")
 	f.BoolVar(&cfg.ShardedQueries, "querier.parallelise-shardable-queries", false, "Perform query parallelisations based on storage sharding configuration and query ASTs. This feature is supported only by the chunks storage engine.")
+	f.Var(&cfg.ForwardedHeaders, "querier.forward-headers-list", "List of headers which should be forwarded by the query_range middleware to downstream")
```

Comment:
People reading this documentation don't know about the code internals.  Remove mention of `query_range middleware`.
`Forwarded` sounds past tense.  What about `ForwardHeaders`?